evap/evaluation/templates/infotext.html

```diff
@@ -1,6 +1,6 @@
 {% load static %}
 
-{% if not infotext.is_empty %}
+{% if infotext and not infotext.is_empty %}
```

Comment:
If we don't pass an info text, we simply should not include this template, should we?